### PR TITLE
Milestone 2: App

### PR DESCRIPTION
**Pull request by Lizdev-05**
In this milestone, I achieved the following...

- Created a greeting model
- Created an API route that provides a random greeting

### CODE REVIEW

**Comment by sja-thedude** [comment]
Hi @Lizdev-05,
Kindly add the readme file, if you could add in 10 mins then I will proceed with the review else invalidate it. Also, make sure to keep the below points in mind.
1- In the README file for the front end, students should provide a link to the Rails API (backend app).
2- In the README file for the backend, you should find a link to the frontend part.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @Lizdev-05, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have a readme file. Also, make sure to keep the below points in mind.
1- In the README file for the front end, students should provide a link to the Rails API (backend app).
2- In the README file for the backend, you should find a link to the frontend part.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Lizdev-05** [comment]
@sja-thedude, thank you for reviewing my project but I have a ReadMe on my project and the links are present.
Kindly check the images below 👇
Screenshot 2022-11-22 at 21 22 42

**Comment by Lizdev-05** [comment]
Kindly check the client part of the Readme, I have the link there.
Screenshot 2022-11-22 at 21 24 39

File: config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+  resources :greetings, only: [:index]
```
Comment (FrederickMih) [inline config/routes.rb:6]: - [**Optional**] As a general rule of thumb, let's always version our API. Versioning allows developers to improve their API in the future without breaking the client's application when new updates are available.
- Hint

```.ruby
 namespace :api do
    namespace :v1 do
      resources :greetings, only: [:index], defaults: { format: 'json' }
    end
  end
```

You'll also need to modify the controller path api/V1

**Review by FrederickMih — APPROVED** [review]
Hi @Lizdev-05, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Correctly set up an API for returning random greetings :+1: 
- The React app consumes the Rails API and the two apps work together :100: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**